client/question/components/LearnerViewDirective.js

```diff
@@ -590,6 +602,13 @@ tie.directive('learnerView', [function() {
          */
         $scope.codeChangeLoopPromise = null;
 
+        /**
+        * Keeps track of whether a current code submission is being run.
+        *
+        * @type {boolean}
+        */
+        $scope.processing = false;
```

Comment:
This name is too vague. Perhaps something like $scope.submissionIsBeingProcessed?

Reply:
Hm, I'll try to take a deeper look tomorrow morning as well (seems to already work as intended when I also repulled from master) . The original reasoning behind this change was because I'm able to spam the "Get Feedback" button on my desktop local and server versions (as well as the official TIE link). I originally thought that it had to do with the isBalloonPending function not updating fast enough.